partiql-logical/src/lib.rs

```diff
@@ -1,14 +1,58 @@
 use partiql_value::{BindingsName, Value};
-use petgraph::prelude::StableGraph;
-use petgraph::Directed;
 use std::collections::HashMap;
 
+#[derive(Debug, Hash)]
+pub struct OpId(usize);
+
+impl OpId {
+    pub fn index(&self) -> usize {
+        self.0
+    }
+}
+
+impl PartialEq<Self> for OpId {
```

Comment:
can probably use `#[derive(PartialEq, Eq)]`

Reply:
Yeah I think what confused me was a compiler error that I got during dev; let me take another stab at it.